src/src/rx-serial/SerialHoTT_TLM.cpp

```diff
@@ -348,13 +350,13 @@ void SerialHoTT_TLM::sendCRSFrpm(uint32_t now, HoTTDevices device)
 
     crsfRpm.p.source_id = device;
 
-    uint8_t payloadSize = 1 + 2 * 3;
+    uint8_t payloadSize = 1 + 3 * 2;
```

Comment:
It would be nice to have some constants for the `1` and field size numbers, like the `3` could be `SIZE_24BIT`
Or something like that.